lib/mi_bridges/driver/more_about_self_employment_page.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class MoreAboutSelfEmploymentPage < BasePage
+      TITLE = /More About .*'s Self-Employment/
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        members.each do |member|
```

Comment:
if we have a member who is NOT self employed, will those code error (bc looking for fields that do not exist)?

Reply:
I don't think this will error since it just won't find the labels associated with the member's name.